### PR TITLE
[IDWA-OCR-95] ocr to phdc conversion logic for patient

### PR DESCRIPTION
**Pull request by arinkulshi-skylight**
# Pull Request
https://github.com/CDCgov/IDWA/issues/95
## Description
Integrate the output of OCR into NBS, by generating a PHDC document.

Acceptance Criteria

Map the OCR key/value output into a PHDC input dataclass.
Generate PHDC using the dibbs phdc library, feel free top copy over the code for right now [[link](https://github.com/CDCgov/phdi/tree/8a105ca7e2c495adec9db7a7c8516bbc247df669/containers/message-parser/app/phdc)]
Integrate patient dataclass only

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [x] The code follows best practices and conventions.
- [x] The changes implement the desired functionality or fix the reported issue.
- [x] The tests cover the new changes and pass successfully.
- [x] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: OCR/ocr/services/phdc_converter/ocr_phdc_converter.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+from .builder import PHDCBuilder
+from .models import PHDCInputData, Patient, Name, Address, Telecom
+
+
+class OCRPHDCConverter:
```
Comment (zdeveloper) [inline OCR/ocr/services/phdc_converter/ocr_phdc_converter.py:5]: rename this to remove the OCR since it doesnt really do OCR stuff

**Review by zdeveloper — DISMISSED** [review]
LGTM! thank you

**Review by knguyenrise8 — APPROVED** [review]
